config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
 
   resources :audio_files, only: [:new, :create, :edit, :update]
+  resources :charges
```

Comment:
Do you think it's better to explicitly limit the routes exposed? I kinda like that it's clearer without having to look back at the actual controller
ie: `resources :charges, only: [:new, :create]`

Reply:
Good plan. Also thinking about renaming to "payments" instead of "charges".